VideoLocker/res/layout/panel_find_course.xml

```diff
@@ -1,74 +1,55 @@
-            android:background="@drawable/edx_brand_button"
-            android:text="@string/find_course_btn_text"
-            android:textAllCaps="true"
-            android:textColor="@color/white"
-            android:textSize="15sp" />
-        
-        <TextView
-            android:id="@+id/course_not_listed_tv"
-            style="@style/semibold_text"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginBottom="10dp"
-            android:layout_marginTop="17dp"
-            android:layout_below="@id/course_btn"
-            android:layout_centerHorizontal="true"
+            android:background="@drawable/bg_panel_find_courses"
+            android:descendantFocusability="blocksDescendants"
             android:gravity="center"
-            android:linksClickable="true"
-            android:text="@string/course_not_listed_text"
-            android:textColor="@color/text_navigation"
-            android:textSize="14sp"
-            android:contentDescription="@string/course_not_listed_btn" />
-        
-    </RelativeLayout>
-    <!-- put a frame over this cell, so that corners look rounded -->
+            android:orientation="vertical">
 
-    <ImageView
-        android:id="@+id/frame"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_alignBottom="@id/panel_course_layout"
-        android:layout_alignLeft="@id/panel_course_layout"
-        android:layout_alignRight="@id/panel_course_layout"
-        android:layout_alignTop="@id/panel_course_layout"
-        android:background="@drawable/bg_frame_course" />
+            <TextView
+                android:id="@+id/findcourse_tv"
+                style="@style/semibold_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/edx_margin"
+                android:layout_marginBottom="@dimen/edx_margin"
+                android:gravity="center"
+                android:text="@string/find_course_text"
+                android:textColor="@color/edx_grayscale_neutral_x_dark"
+                android:textSize="@dimen/edx_base" />
+
+            <Button
+                android:id="@+id/course_btn"
+                style="@style/edX.Widget.BrandPillButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/find_course_btn_text" />
+
+            <TextView
+                android:id="@+id/course_not_listed_tv"
+                style="@style/semibold_text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/widget_margin"
+                android:contentDescription="@string/course_not_listed_btn"
+                android:gravity="center"
```

Comment:
It is needed to ensure the text is centered (in case the string is changed/translated to be long enough that it has to wrap).